src/frontend/components/singlePost.jsx

```diff
@@ -0,0 +1,66 @@
+import { AiOutlineRetweet, AiFillLike } from "react-icons/ai";
+import { MdModeComment, MdTipsAndUpdates } from "react-icons/md";
+import { BsThreeDots } from "react-icons/bs";
+import { FaShare } from "react-icons/fa";
+import {usePosts} from "frontend/context/postContext";
+
+const SinglePost = () => {
+    const {postsState: {posts}, receivedPost} = usePosts();
+
+    return (<>
+    {posts.map(post => {
+                return <div key={post._id} className="flex border-b border-grey px-4 py-4">
+                    <img src="assets/spectrum.png" alt="profile pic" className="rounded-full h-14 w-14 mr-3 cursor-pointer" title="profile pic" />
```

Comment:
I think this post has been named wrongly as it is mapping over posts array.

Reply:
Done